cmd/kubelet/app/server.go

```diff
@@ -545,7 +545,7 @@ func getReservedCPUs(machineInfo *cadvisorapi.MachineInfo, cpus string) (cpuset.
 		return emptyCPUSet, nil
 	}
 
-	topo, err := topology.Discover(machineInfo)
+	topo, err := topology.Discover(klog.Background(), machineInfo)
```

Comment:
Would it make sense to pass context from the caller to this function instead of creating a new one?

Reply:
Yes. Typically, requiring a context or logger in some "leaf" function trickles all the way up to main (in a binary) or up to the `Test*` unit test function.

If you want to split up that work across different PRs, then use `klog.TODO()` as an indication that this isn't the final code.

Reply:
it's simpler: I haven't noticed the parent context. Trivial overlook on my side. Will fix (and be more careful next time)